src/lib.rs

```diff
@@ -55,7 +54,7 @@ impl Rollback {
 
 /// Provides unique ids for your Rollback components.
 /// When you add the GGRS Plugin, this should be available as a resource.
-#[derive(Default)]
+#[derive(Resource, Default)]
```

Comment:
```suggestion
#[derive(Resource, Default)]
#[reflect(Resource)]
```

Reply:
What does this accomplish ? wouldn't Resource already have some reflect capabilities ? or is it to enable bevy_egui reflection ?

Reply:
Hmm.. maybe it's not needed? I think I maybe just made a brainfart and thought you added `derive(Reflect)`.

I guess the real answer is to just check what works :P

Reply:
Yep, sorry about the noise. We could perhaps add reflect in another PR. It might be useful for debugging.